projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -63,8 +63,7 @@ public static function instance() {
 	public function load_php() {
 		$this->base_load_php();
 
-		require_once __DIR__ . '/class-jetpack-search-settings.php';
-		new Jetpack_Search_Settings();
+		new Automattic\Jetpack\Search\Settings();
```

Comment:
Is this needed now that the class will be autoloaded?

Reply:
It's needed as the class needs to be initialized, which added some actions/filters. Thanks